### DISCUSSION TITLE
Job is not executing when I pass the jobData

### DISCUSSION THREAD

**Discussion by sanjaysingharcadix** [quartznet/quartznet#1763] [Q&A]
Hi ,

When I execute the job without any parameter(Job Data ) it is working fine. But  when I pass job data it is not executing job.

 //Create job
            IJobDetail commonJob = JobBuilder.Create<SmallJobs>()
                                        .UsingJobData("objJobData", "sample name")
                                      .WithIdentity(objParam["strJobName"].ToString())
                                      .RequestRecovery(true)
                                      .Build();
            
            //Create Trigger
            ITrigger commonTrigger = TriggerBuilder.Create()
                                             .WithIdentity(objParam["strTriggerName"].ToString())
                                             .StartNow()
                                             .Build();
          
            _quartzScheduler.ScheduleJob(commonJob, commonTrigger);

**Comment by lahma** [comment]
What do the logs say?

**Reply by sanjaysingharcadix** [reply]
I didn't implement log with Quartz.

**Reply by lahma** [reply]
Then you should, they will tell you information.

**Reply by sanjaysingharcadix** [reply]
Hi lahma, I tried with log but log didn't show any error while executing job with job parameter , Is there any other way to find out the issue.

**Reply by lahma** [reply]
You guess is as good as mine without the logs that Quartz produces, sorry.
